docs/content/preview/explore/change-data-capture/using-logical-replication/overview.md

```diff
@@ -9,4 +9,195 @@ menu:
+
+### PUBLICATION
+
+A publication is a set of changes generated from a table or a group of tables, and might also be described as a change set or replication set. Each publication exists in only one database.
+
+Publications are different from schemas and do not affect how the table is accessed. Each table can be added to multiple publications if needed. Publications may currently only contain tables. Objects must be added explicitly, except when a publication is created for ALL TABLES.
+
+### OUTPUT PLUGIN
+
+Output plugins are the components used to decode WAL changes and transform them into a specific format that can be consumed by replication clients. These plugins are notified about the change events that need to be processed and sent via various callbacks. These callbacks are only invoked when the transaction actually commits.
+
+YugabyteDB supports the following four output plugins:
+* `yboutput`
+* `pgoutput`
+* `test_decoding`
+* `wal2json`
+
+All these plugins are pre-packaged with YugabyteDB and do not require any external installation.
+
+### LSN
+
+LSN in YugabyteDB is an unsigned 64-bit integer that uniquely identifies a change record or a transaction boundary record that is consumed from a given replication slot.
+
+In YugabyteDB, LSN values from different slots are considered unrelated and should not be compared. In YugabyteDB, LSN no longer represents the byte offset of a WAL record. 
+
+LSN values for a single replication slot satisfy the following properties: 
+
+- **Uniqueness** 
+
+  - LSN values for the change and transaction boundary records for a given replication slot are unique. In particular, changes from different tablets of the same table will have unique LSN values.
+
+* **Ordering** 
+
+  - LSN values can be compared ( <, >, = ).
+
+  - The LSN of the BEGIN record of a transaction will be strictly lower than the LSN of the changes in that transaction that in turn will be strictly lower than the LSN of the COMMIT record of the same transaction.
+
+  - The LSNs of change records within a transaction will be in increasing order and will correspond to the order in which those changes were made in that transaction. That is, the LSN of an earlier change will have a strictly lower value than the LSN of a later change within the same transaction. This is the case even if the changes correspond to rows in different tablets of the same or different tables.
+
```

Comment:
For a given replication slot, the LSN of a COMMIT record of an earlier transaction will be strictly lower than the LSN of the COMMIT record of a later transaction.

Reply:
@asrinivasanyb can you confirm if the content you have shared is a replacement for the highlighted point or the next point?

Reply:
This should be replacement of 
"The LSN of the commit record of a transaction will be strictly lower than the LSN of the BEGIN record of  a transaction with greater commit time."